[INTEL_HPU] enabled fastdeploy PR testing

FocusLuo opened the pull request:
enabled fastdeploy PR testing on INTEL_HPU

FocusLuo commented:
depended on the issue fixing of https://github.com/PaddlePaddle/FastDeploy/issues/4552

plusNew001 commented on scripts/run_ci_hpu.sh:
```diff
@@ -0,0 +1,145 @@
+if [  $ENABLE_TESTING_ERNIE45_21B_A3B_Paddle -eq 1 ]; then
+    export model_path=${MODEL_PATH}/ERNIE-4.5-21B-A3B-Paddle/
+    export HPU_VISIBLE_DEVICES=2
+    FD_ENC_DEC_BLOCK_NUM=8 HPU_PERF_BREAKDOWN_SYNC_MODE=1 HPU_WARMUP_BUCKET=1 HPU_WARMUP_MODEL_LEN=4096 FD_ATTENTION_BACKEND=$FD_ATTENTION_BACKEND_NAME python -m fastdeploy.entrypoints.openai.api_server --model $model_path --port $FD_ENGINE_QUEUE_PORT --engine-worker-queue-port $FD_ENGINE_QUEUE_PORT --metrics-port $FD_METRICS_PORT --num-gpu-blocks-override 3072 --kv-cache-ratio 0.98 --num-gpu-blo ...
```
这个位置也应该重定向到server.log 后台运行吧

plusNew001 commented on scripts/run_ci_hpu.sh:
```diff
@@ -0,0 +1,145 @@
+python -m pip uninstall fastdeploy_intel_hpu -y
+
+#to install paddlepaddle
+pip install paddlepaddle==3.1.1
```
这里我们一般ci会使用nightly的包或者最新release分支的包，以保证用户高可用性

FocusLuo commented:
docker_image用的是这个 cccr-2vdh3abv-pub.cnc.bj.baidubce.com/device/paddle-hpu:latest
是否需要在scripts/run_ci_hpu.sh中加上build and install Intel HPU PaddleCustomDevice?

FocusLuo commented:
based on reviewing feedback, updated 2nd:
1, jobs:
  CI_HPU:
    runs-on:
      group: HPU
2, fixed a typo issue 
env:
          docker_image: ccr-2vdh3abv-pub.cnc.bj.baidubce.com/device/paddle-hpu:latest
==> cccr-2vdh3abv-pub.cnc.bj.baidubce.com/device/paddle-hpu:latest
3, add paddlecustomdevice binary install
#to install paddlecustomdevice? (paddle-intel-hpu)
pip install https://paddle-qa.bj.bcebos.com/HPU/paddle_intel_hpu-0.0.1-cp310-cp310-linux_x86_64.whl

FocusLuo committed:
[INTEL HPU] added hpu ci work flow support

Signed-off-by: Luo, Focus <focus.luo@intel.com>

FocusLuo committed:
[INTEL HPU] added run ci hpu test scripts

Signed-off-by: Luo, Focus <focus.luo@intel.com>

FocusLuo committed:
[INTEL HPU] enabled HPU ernie test case

Signed-off-by: Luo, Focus <focus.luo@intel.com>

FocusLuo committed:
[INTEL HPU] updated Intel Gaudi Readme with Warmup disable cmdline

Signed-off-by: Luo, Focus <focus.luo@intel.com>

plusNew001 committed:
Merge branch 'develop' into pr_testing_intel_hpu

plusNew001 committed:
Modify paddlepaddle installation command

Updated paddlepaddle installation command to use a specific index URL.

plusNew001 committed:
Update run_ci_hpu.sh

plusNew001 committed:
Rename json directory to nlohmann_json

Rename extracted json directory to nlohmann_json.

plusNew001 committed:
Update ci_hpu.yml

plusNew001 committed:
Set pip global index URL to Tsinghua mirror

plusNew001 committed:
Update CI workflow to use self-hosted runner and paths

plusNew001 committed:
Update Docker image in CI workflow

plusNew001 committed:
Modify HPU installation URLs in run_ci_hpu.sh

Updated the installation URL for paddle_intel_hpu and added paddlenlp_ops installation.

plusNew001 committed:
Fix paddle_intel_hpu installation URL

Corrected the URL for paddle_intel_hpu wheel installation.
